frontend/index.html

```diff
@@ -95,15 +96,15 @@ <h4 style="font-weight: lighter;">
             </td>
 
             <td style='padding:60px'>
-              <img src="images/maintenance.svg" alt="Hammer and Wrench">
+              <img src="images/maintenance.svg" longdesc="maintenancemode" alt="Hammer and Wrench">
```

Comment:
Same as above or maybe it should be "maintenanceMode"?

Reply:
I notice you changed some longdesc to have spaces. Should this one get spaces?

Reply:
If we're using camelcase on longdesc then this should be maintenanceMode